product_docs/docs/edbcloud/beta/getting_started/03_create_cluster.mdx

```diff
@@ -22,12 +22,11 @@ To create a cluster:
 10. Select the **Instance Type** based on the number of vCPUs and amount of memory desired.
 11. Select the **Storage** type and amount of storage needed for your cluster.
    !!! Note
-       Azure premium storage is the only option for the Preview.
+       Azure premium storage is the only option.
```

Comment:
Kelly and I think you should keep "For the Preview" unless it is not changing for GA.

Reply:
@aswright491 suggested dropping the "for the preview."

Reply:
It might change for GA.